### ISSUE TITLE
Cannot login to the VM using guest Session

### ISSUE THREAD

**Issue by eipVCCRI** [CobraLab/MINC-VM#23]
Hi there,

I'm looking to use Pydpiper for mouse image registration, and was hoping to use the VM setup, so I don't have to go thru the difficult process of install all the tools. 

However when it ask for a sign on id, I try to to use the Guest Session option to log in, it just pops back to the log in request.

![login](https://user-images.githubusercontent.com/33052080/34475459-67d8a3a8-efe0-11e7-98cf-a223f74963b3.JPG)

Hope you can help.
Eddie Ip

**Comment by gdevenyi** [comment]
Hi, it should be auto logging in. Otherwise, the password for the minc user is minc

**Comment by eipVCCRI** [comment]
Hi Gabriel,

I tried the password for MINC user, but still just gives pops back to login request.

I'm using image artifact version 506, which was created on the 2nd Jan 2018.

Hope you can help.
Eddie

**Comment by gdevenyi** [comment]
Thanks for the details, I'll download that version and give it a test.

**Comment by gdevenyi** [comment]
Okay, problem located.

Something has changed in the anaconda config which breaks the graphical login. I'm sourcing anaconda in the login profile which is the wrong thing (tm) anyways. I'll fix this with the autobuild, but here's how you can fix yours.

In virtualbox, "Right Ctrl + F1" to get to a terminal login.

Username/password minc/minc
```
> sudo -i
pw: minc
> nano /etc/profile
#Scroll to bottom, comment out or delete 'source /opt/miniconda/bin/activate'
```

**Comment by gdevenyi** [comment]
Fixed by commit 14939288657cd1f80c9a03c9145568f3baad93cd

**Comment by eipVCCRI** [comment]
Thanks Gabriel, the fix you gave me worked on my version. I look forward to the next build.

Thanks. Eddie
